webapp/src/ts/modals/training-cards/training-cards.component.html

```diff
@@ -0,0 +1,17 @@
+<mm-modal class="enketo-modal content-pane" id="id" [titleKey]="modalTitleKey | translate" (onCancel)="cancel()" [hideFooter]="hideModalFooter">
+
+  <div class="item-content empty-selection" *ngIf="loadingContent">
+    <div><div class="loader"></div></div>
+  </div>
+
+  <div class="item-content empty-selection" *ngIf="!loadingContent && contentError">
+    <div>{{ errorTranslationKey | translate }}</div>
+  </div>
+
+  <div class="item-content" [hidden]="loadingContent || contentError">
+    <div class="body">
+      <mm-enketo [formId]="formWrapperId" [status]="enketoStatus" (onSubmit)="saveForm()" (onCancel)="cancel()"></mm-enketo>
+    </div>
+  </div>
+
+</mm-modal>
```

Comment:
Does this have to be a modal? Consider having a modal which has a button that links to a page for training. That way enketo will just work and the user will have the full size to work in. Plus the page will have a URL so it can be refreshed, bookmarked, or sent to users via SMS.

Reply:
Yes, Care Teams considered pages, but at the end it was designed with modals and tested with CHWs from i-tech and d3.

Reply:
A couple of reasons why using modals considering our users are:
- it helps them focus on a single self-contained task, we need them to complete the trainings for the success of upcoming new changes. They will have a cancel button in case they want to postpone the training, but they will not have other distractions available (tabs, hamburger menu) because the grey overlay. 
-  they remain in the same context they spect to be, and avoid getting them feeling lost because they got navigated to another page that will have different content every time. 

When using mobile, the space available to display form's elements in modal is very good, not too different when using page. But we can revisit this in case any element looks bad once we have the trainings designs.

I'm not sure if we want to bookmark trainings, specially if the training is retired because no longer relevant or it was completed before, or if we need them to focus in another training. Also trainings can have a strategic order, we can control with the start_date 

I'm tagging @n-orlowski and @michaelkohn in case we want to revisit some of these ideas or provide more info :)

Reply:
All good points, and I realise this is late in the piece for a design change. I'm happy to follow up with a larger group if you think that'd help.

Reply:
Looks like we're keeping modals, but please feel free to open a discussion in the care team channel! Thanks!